pkg/api/azenvtypes.go

```diff
@@ -182,7 +182,7 @@ var (
 		ImageOffer:     "aks-windows",
 		ImageSku:       "2019-datacenter-core-smalldisk-2006",
 		ImagePublisher: "microsoft-aks",
-		ImageVersion:   "17763.1217.200603",
+		ImageVersion:   "17763.1282.200610",
 	}
 
 	// WindowsServer2019OSImageConfig is the 'vanilla' Windows Server 2019 image
```

Comment:
Can you update this one to use the same ImageVersion the aks-windows VHD was based off of?

Reply:
For the vanilla windows server the June patches haven't been released as an image yet.  So I believe does match: https://github.com/Azure/aks-engine/pull/3439/files#diff-d61a27407ad48d7a4154938c47584c43

Reply:
Wow - yea.
I've never seen the -with-containers image not get released at the same time as the vanilla WS2019 images..